rpm/SPECS/iotaul.spec

```
@@ -170,6 +170,19 @@ fi
 %{_install_dir}
 
 %changelog
+* Fri Apr 29 2022 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 1.21.0
+- Add: apply expression and payload transformations to commands (iota-json#634, iota-json#627)
+- Fix: ensure command QoS for MQTT is an integer
+- Fix: ensure mqtt client_id is unique between reconnections to avoid reconnection loop (iota-json#650)
+- Fix: bad mqtt measure is progressing as a multiple measure after be procesed as hex string
+- Fix: search device and group for a command using entity type when provided to proper match (iota-node-lib#1211)
+- Fix: https:// url instead of git:// for npm dependences
+- Remove: obsolete iotaULTester binary
+- Upgrade moment dep from 2.27.0 to 2.29.2 due to security vulnerability (CVE-2022-24785)
```

```suggestion
- Upgrade iotagent-node-lib dependency from 2.19.0 to 2.21.0
- Upgrade moment dep from 2.27.0 to 2.29.2 due to security vulnerability (CVE-2022-24785)
```

Note we step two iotagent-node-lib versions, given 2.20.0 was created for the 1.20.1 hotfix